Translations/pt_BR.po

```diff
@@ -10021,7 +10189,7 @@ msgstr "Descer para a cripta"
 
 #: Source/trigs.cpp:371
 msgid "Down to Hive"
-msgstr ""
+msgstr "Descer para a colmeia"
```

Comment:
```suggestion
msgstr "Descer para a Colméia"
```

Reply:
I was wrong, the only problem here is the capitalization, I've updated my comment.